stave/templates/stave/officiating_history.html

```diff
@@ -0,0 +1,103 @@
+{% extends "base.html" %}
+{% load stave_tags %}
+
+{% block content %}
+
+<h2>Officiating History</h2>
+
+<table>
+    <thead>
+        <tr>
+            <td>
+                Date
+            </td>
+            <td>
+                Event Name
+            </td>
+            <td>
+                Event Location
+            </td>
+            <td>
+                Event Host
+            </td>
+            <td>
+                Home / High Seed
+            </td>
+            <td>
+                Visitor / Low Seed
+            </td>
+            <td>
+                Association
+            </td>
+            <td>
+                Game Type
+            </td>
+            <td>
+                Position
+            </td>
+            <td>
+                2nd Position
+            </td>
+            <td>
+                Software
+            </td>
+            <td>
+                Head Referee
+            </td>
+            <td>
+                Head NSO
+            </td>
+        </tr>
+    </thead>
+    <tbody>
+        {% for crew_assignment in object_list %}
+        {% with gh=crew_assignment|game_history %}
+        <tr>
+            <td>
+                {# TODO: This should use the local timezone. #}
+                {{ gh.game.start_time|date:"Y-m-d" }}
+            </td>
+            <td>
+                {{ gh.event.name }}
+            </td>
+            <td>
+                {{ gh.event.location }}
+            </td>
+            <td>
+                {{ gh.event.league.name }}
+            </td>
+            <td>
+                {# Home / High Seed #}
```

Comment:
I'm working on revamping the editing UI for Event + Game, as part of which I'll add actual fields for these, association, and game type.

Reply:
Just merged, will add these new fields here.